src/components/Common/UI/Switch/Switch.module.scss

```diff
@@ -0,0 +1,98 @@
+:global(.react-aria-Switch) {
```

Comment:
Nit: for encapsulation we might consider placing this within a parent `.root` class or similar, otherwise it'll end up being truly global scope (in which case we'd probably want to just keep it at the global level)